### PR TITLE
Remove deprecated param and flexibility for prompt 

### PR DESCRIPTION
**Pull request by toshish**
- **Description:** Updated to remove deprecated parameter penalty_alpha, and use string variation of prompt rather than json object for better flexibility.  - **Issue:** the issue # it fixes (if applicable),
  - **Dependencies:** N/A
  - **Tag maintainer:** @eyurtsev
  - **Twitter handle:** @symbldotai

### CODE REVIEW

**Comment by toshish** [comment]
hey @eyurtsev @baskaryan can you please review and merge the PR if everything looks good? Thanks.

**Comment by toshish** [comment]
@baskaryan @eyurtsev reminder, to review and merge this.

**Comment by toshish** [comment]
Hey @hwchase17 @baskaryan @eyurtsev can you please merge this PR? It's been pending for over 2 weeks now. Thanks!

File: libs/langchain/tests/integration_tests/llms/test_symblai_nebula.py [hunk 1]
```diff
@@ -40,8 +40,8 @@ def test_symblai_nebula_call() -> None:
 
     instruction = """Identify the main objectives mentioned in this 
 conversation."""
-    prompt = PromptTemplate.from_template("{instruction}\n{conversation}")
-
+    prompt = PromptTemplate.from_template(template="{instruction}\n{conversation}")
     llm_chain = LLMChain(prompt=prompt, llm=llm)
     output = llm_chain.run(instruction=instruction, conversation=conversation)
+    print(output)
```
Comment (hwchase17) [inline libs/langchain/tests/integration_tests/llms/test_symblai_nebula.py:46]: stray print statement

**Review by hwchase17 — APPROVED** [review]

**Comment by toshish** [comment]
@hwchase17 thanks for the correction. Any eta on when the PR will be merged?

**Comment by hwchase17** [comment]
merging now! sorry for delay

**Pull request merged**